02_what_why_intro.Rmd

```diff
@@ -0,0 +1,20 @@
+# What is Data Visualization?
+
+```{r, include = FALSE}
+ottrpal::set_knitr_image_path()
+```
+
+## Learning Objectives
+
+This chapter will cover:
+
+- {LO1}
+- {LO2}
+
+## What is Data Visualization?
+
+### Exploratory Data Analysis (EDA)
```

Comment:
I do like the exploratory vs explanatory dichotomy for explaining the aims of visualization: https://laderast.github.io/data_storytelling_bdc/#5